python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -192,7 +193,7 @@ async def handle_summary_state(self):
     async def configure(self, config):
         self.config = config
 
-    def _compute_averages_and_send_telemetry(self):
+    def _compute_median_values_and_send_telemetry(self):
```

Comment:
Please consider a more generic name such as `_compute_statistics_and_send_telemetry`. This would allow you to add or change statistics.

Reply:
Done.